test/java/seedu/addressbook/util/TestUtil.java

```diff
@@ -75,4 +77,18 @@ public static void assertTextFilesEqual(Path path1, Path path2) throws IOExcepti
         List<String> list2 = Files.readAllLines(path2, Charset.defaultCharset());
         assertEquals(String.join("\n", list1), String.join("\n", list2));
     }
+
+    /**
+     * Executes the command, and asserts the result message is as expected.
+     */
+    public static void assertCommandResult(Command command, AddressBook addressBook, AddressBook expectedAddressBook,
+                                                                                               String expectedMessage) {
```

Comment:
@damithc So, I do wonder whether this method is doing too much to be considered a "utility" method. Of the four arguments, the 1st is used in the 1st statement, the 4th is used in the 2nd statement, and the 2nd and 3rd are used in the 3rd statement. Note how the use of these arguments do not overlap at all.